src/flux/hub.ts

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
```

Comment:
I've moved these logic to `services/hubAPI` in #143. I think we can remove it here.

Reply:
Cool, after we merge yours I will refactor this reflect the same structure